### ISSUE TITLE
add doc for how to configure a metric with Prometheus

### ISSUE THREAD

**Issue by lust4life** [logary/logary#413]
@lust4life I've added a section in the README so people know how to configure a metric with Prometheus. Perhaps you could add one of the samples from this thread there?

_Originally posted by @haf in https://github.com/logary/logary/pull/395#issuecomment-479073948_

**Comment by haf** [comment]
I created a CD pipeline with google cloud builder and next.js; just edit the docs in `./docs` and commit, and it'll be automatically deployed at https://logary.tech/ 2 minutes later.

**Comment by haf** [comment]
The site is now live and up to date with all docs that we have!

**Comment by 64J0** [comment]
The site is down for me @haf

**Comment by 64J0** [comment]
Maybe we could start using [Nacara](https://github.com/MangelMaxime/Nacara) to build the next version of the docs.

**Comment by haf** [comment]
Yes possibly, or preferrably a generator that is mainstream and supports a library version selector.

**Comment by haf** [comment]
https://docusaurus.io/ is warmly recommended by some of my CTO friends

**Comment by haf** [comment]
We'll be documenting Logary-JS with this as well.
